auter.aptModule

```diff
@@ -103,6 +103,7 @@ echo "This message should not print"
       fi
 
       logit "INFO: Updates complete (${PACKAGE_MANAGER} ${TRANSACTIONID}). You may need to reboot for some updates to take effect"
+      log_last_run
```

Comment:
I would move this to the first line of the apply_updates function. That way you will get this log message regardless of the result of the apply function (successful, failed or no updates available).